R/bbox.R

```diff
@@ -62,7 +62,12 @@ bbox.MtrxSetSetSet = function(obj) {
 #' @examples
 #' a = st_sf(a = 1:2, geom = st_sfc(st_point(0:1), st_point(1:2)), crs = 4326)
 #' st_bbox(a)
-#' st_as_sfc(st_bbox(a))
+#' b = st_as_sfc(st_bbox(a))
+#' b_larger = b * 1.1
```

Comment:
this functionality has nothing to do with st_bbox: why introduce it here?

Reply:
To show people how you can work with bbs to meet a common need: to increase or decrease the 'field of view'. In short: user-friendliness.

Reply:
But agree this example may not best demonstrate that (hence it being work in progress - just an idea).